collections/guides/use/reveal-card-data-with-elements.md

```diff
@@ -0,0 +1,169 @@
+import { CardElement as ICardElement } from '@basis-theory/basis-theory-js/types/elements';
+
+export const DisplayCard = () => {
+  const { bt } = useBasisTheory();
+  const cardRef = useRef<ICardElement>(null);
+
+  return (
+    <CardElement id="myCard" ref={cardRef}/>
+  );
+};
+```
+
+### Using the Expiring Application Key to Retrieve a Token with Elements
+
+Assuming your client code is already fetching `expiring` application keys as per previous steps, these can then be used to retrieve a token value, granted that proper permissions are applied. 
+Any attribute from the token like `metadata` can be directly accessible from the response, <strong>except</strong> for `data`, which represents just a reference to the actual data value that can be used with `setValue` in the next step.
+
+```jsx
+const cardToken = await bt.tokens.retrieve(tokenId, { apiKey: expiringKey });
+```
+
+### Revealing Token Data with Elements
+
+As said previously, the retrieved token `data` value does not contain the actual data but just a reference to it. The actual `data` is safely stored in a Basis Theory `iframe` and can be set as the value for an Element input internally using the `setValue` method:
+
+```jsx
+cardRef.current.setValue(cardToken.data)
+```
+
+### Putting it All Together
+
+See the complete code we created in this tutorial below. 
+
+```jsx
+import { useRef } from 'react';
+import { CardElement, useBasisTheory } from '@basis-theory/basis-theory-react';
+import { CardElement as ICardElement } from '@basis-theory/basis-theory-js/types/elements';
+
+export const DisplayCard = () => {
+  const { bt } = useBasisTheory();
+  const cardRef = useRef<ICardElement>(null);
+
+  const revealCard = async () => {
+
+    // fetching the expiring application key from your server side code
+    const {
+      data: { expiringKey },
+    } = await axios.post('/api/authorize');
+
+    // using the expiring application key to retrieve the token
+    const cardToken = await bt.tokens.retrieve(tokenId, { apiKey: expiringKey });
+
+    // setting the token data value into the CardElement
+    cardRef.current.setValue(cardToken.data); 
+  }
+
+  const cardToken = await bt.tokens.retrieve(tokenId, { apiKey: expiringKey });
+
+  return (
+    <>
+      <CardElement id="myCard" ref={cardRef}/>
+      <button type="button" onClick={revealCard}>Reveal Card</button>
+    </>
+  );
+};
+```
+
+That's it, now the token data can be displayed to the user 🎉, and since your client does not have direct access to the actual data value, it falls out of compliance scope.
+
+## Demo Repository
+
+To better demonstrate this feature, a demo [repository](https://github.com/Basis-Theory-Labs/display-card-example) that you can clone and run on your local machine has been made available.
+
+The repository uses a few of the [Element's user experience](https://docs.basistheory.com/elements/#introduction) features to better enhance the usage experience and to make testing easier like:
```

Comment:
I'm not really sure what was intended by this sentence - 
1. the demo repository showcases some best practices for working with Elements
2. OR the demo repository showcases how you can build a nice user experience for your end users?
3. OR the demo repository showcases how you can build abstractions around Elements that make the developer experience better? 

Assuming the first interpretation, I took a stab at improving the wording:
```suggestion
This repository showcases some best practices and techniques for simplifying working with [Elements](https://docs.basistheory.com/elements/#introduction), including:
```